kythe/java/com/google/devtools/kythe/extractors/shared/IndexInfoUtils.java

```diff
@@ -66,8 +80,38 @@ public static CompilationDescription readIndexInfoFromStream(InputStream inputSt
     return new CompilationDescription(compilationUnit, fileContents);
   }
 
-  public static void writeIndexInfoToStream(CompilationDescription description, OutputStream stream)
-      throws IOException {
+  public static Collection<CompilationDescription> readKZip(String path)
```

Comment:
This is unused.  Meant to be in a later commit?

Reply:
yeah, its for the next change.